### ISSUE TITLE
[v1] QDialog's maximized + position don't work together anymore

### ISSUE THREAD

**Issue by tegola** [quasarframework/quasar#3769]
### Software version
```
$ quasar info

Operating System - Darwin(17.7.0) - darwin/x64
NodeJs - 9.11.1

Global packages
  NPM - 5.6.0
  yarn - 1.15.2
  @quasar/cli - 1.0.0-beta.3
  cordova - 8.1.2 (cordova-lib@8.1.1)

Important local packages
  quasar - 1.0.0-beta.14 -- High performance, Material Design 2, full front end stack with Vue.js -- build SPA, SSR, PWA, Hybrid Mobile Apps and Electron apps, all simultaneously using the same codebase
  @quasar/app - 1.0.0-beta.15 -- Quasar Framework App CLI
  @quasar/extras - 1.1.1 -- Quasar Framework fonts, icons and animations
  vue - 2.6.10 -- Reactive, component-oriented view layer for modern web interfaces.
  vue-router - 3.0.2 -- Official router for Vue.js 2
  vuex - 3.1.0 -- state management for Vue.js
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.3.4 -- Babel compiler core.
  webpack - 4.29.6 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 3.2.1 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - 4.1.1 -- A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.
  register-service-worker - 1.6.2 -- Script for registering service worker, with hooks

Quasar App Extensions
  *None installed*
```

### JsFiddle
https://jsfiddle.net/Lfohsm0x/

### What did you get as the error?
After upgrading to beta 14, all dialogs using `maximized` and `position` don't appear anymore - I can only see the backdrop.

Also, I swear it was working while reproducing on jsFiddle, then thankfully stopped after a while. Looks like I started when the CDN still didn't update to the latest version, then caught it mid update.

### What were you expecting?
Dialog should appear from the specified position (`bottom` in my case).

**Comment by rstoenescu** [comment]
Hi,

This is an incorrect usage. Your Dialog should either be maximized or sticking to one of the sides (up, bottom, left, right). The way I see it, you are interested in the animation only, which in this case you should use `transition-show` and `transition-hide` props.

```html
<q-dialog
    v-model="open"
    persistent
    maximized
    transition-show="slide-up"
    transition-show="slide-down"
  >
```

**Comment by rstoenescu** [comment]
https://jsfiddle.net/rstoenescu/4mu0vz6a/1/

**Comment by tegola** [comment]
Correct, but I'm trying to replicate BottomSheet's look while using a dialog with a containerized layout, so I need it to stay anchored at the bottom fo the screen, hence `position="bottom"`.

I'm using a layout because I need a fixed toolbar, but content should expand to fit the layout content (which can even be a couple lines, a pretty small dialog) and never reach the top of the screen, so I'm using `maximized` and setting a max width of 80vh.

While it might have been incorrect usage, this was possibile with the previous version. Any input on how to achieve the same behavior with the new one?

Thanks

**Comment by rstoenescu** [comment]
Ok, I see a small issue with not being able to override the height/width when using `maximized` prop. **Beta.15** (2-3 days away probably) will have an enhancement regarding this, but you'll need to add a `style="height: ..."` to your containerized QLayout (containerized QLayout always needs a height specified somehow). Example:

```html
<q-dialog
      v-model="layoutBottom"
      maximized
      position="bottom"
    >
      <q-layout container style="height: 50vh">
        <q-header>Header</q-header>
        <q-page-container>
          <q-page padding class="bg-white">
            I should be a maximized dialog sliding from the bottom of the screen
          </q-page>
        </q-page-container>
      </q-layout>
    </q-dialog>
```

**Comment by rstoenescu** [comment]
In any case, there are numerous better ways to put a fixed header or footer etc. Check out: https://v1.quasar-framework.org/vue-components/dialog#Example--Scrollable-dialogs

**Comment by tegola** [comment]
Yeah I already use a `height` property on the layout css. It's exactly what I was using previously to keep the maximized dialog layout from reaching the top side of the screen, but it was a `max-height` because the `position` prop would shrink the layout to its content size, and I only needed to limit its height when the content was too much.

I don't think I can replicate the same behavior with the new version, though. But mine is an edge use case and it's not vital. I have fixed my dialogs so those that need to stay anchored to the bottom use `position="bottom"` and a fixed layout height, while those that don't use `maximized` and `transition-show` and `transition-hide`.

I'm glad you found a bug while helping me out on this.
Thanks for your support :)
